policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/solidarity/amount/qst/base.yaml

```
@@ -0,0 +1,15 @@
+description: Quebec limits its QST component of the solidarity tax credit at ot below this base amount.
```

In general let's spell out acronyms in descriptions
```suggestion
description: Quebec provides this Quebec sales tax credit component of its Solidarity Tax Credit.
```